heketi-centos-ci-tests.sh

```diff
@@ -37,6 +35,9 @@ yum -y install \
 	python-virtualenv \
 	ansible
 
+rpm -iU --force https://releases.hashicorp.com/vagrant/2.2.7/vagrant_2.2.7_x86_64.rpm
```

Comment:
Is 'force' needed here? I realize that these are ultimately "scratch" machines but its a bit of a "code smell"

Reply:
Yes, was using the --force option to get around the rpm/yum error "does not update installed package". Replaced with an explicit check and call install only when not installed.